src/components/Person.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {name}
+    </h2>
+
+    {age && (
+      <p className="Person__age">
+        I am
+        {' '}
+        {age}
+      </p>
+    )}
+
+    {isMarried ? (
+      <p className="Person__partner">
+        My
+        {' '}
+        {sex === 'm'
+          ? `wife's`
+          : `husband's`
+        }
+        {' '}
+        name is
+        {' '}
+        {partnerName}
```

Comment:
```suggestion
        My {sex === 'm' ? `wife's` : `husband's`} name is {partnerName}
```
А почему не написать так?